packages/docs/src/page-configs/ui-elements/infinite-scroll/examples/Default.vue

```diff
@@ -6,6 +6,7 @@
     >
       <va-infinite-scroll
         :load="appendRecordsAsync"
+        :offset="40"
```

Comment:
Why do we need to offset in default demo? I think the default demo should work without additional props.

As I user, I want components "just work".

```suggestion
```

Reply:
returned everything back ))